packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -67,22 +61,22 @@ let serverAddress: Address;
 let serverDestination: Destination;
 let browserAddress: Address;
 let browserDestination: Destination;
-let objectiveSuccededPromise: Promise<void>;
 
-beforeAll(() => {
+beforeAll(async () => {
   provider = new providers.JsonRpcProvider(rpcEndpoint);
   assetHolderContract = new Contract(
     ethAssetHolderAddress,
     ContractArtifacts.EthAssetHolderArtifact.abi,
     provider
   );
   mineOnEvent(assetHolderContract);
+  // TODO: The onSendMessage listener can still be executing
```

Comment:
I think this `interop` test should probably be rewritten using the server `wallet`.